src/components/study-pane.js

```diff
@@ -378,6 +378,21 @@ const StudyPane = (props) => {
 
     const { enqueueSnackbar } = useSnackbar();
 
+    const networkRef = useRef();
+    networkRef.current = network;
+
+    const historyRef = useRef();
+    historyRef.current = history;
+
+    const studyUuidRef = useRef();
+    studyUuidRef.current = studyUuid;
```

Comment:
maybe a little overkill as studyUuid  never change ;)

Reply:
I removed it